code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,179 @@
+@Geofencing 
+Feature: Camara Geofencing Subscriptions API ,v0.3.0 - Operations on subscriptions 
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in geofencing-subscriptions.yaml, version v0.3.0
+
+  Background: Common Geofencing Subscriptions setup
+    Given the resource "{apiroot}/geofencing-subscriptions/v0.3/"  as geofencing base-url                                                           
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+
+  @geofencing_subscriptions_01_Create_geofencing_subscription_for_a_device
+  Scenario:  Create geofencing subscription
+    Given a valid subscription request body 
+    When the  request "createSubscription" is sent 
+    Then the response code is 201
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_02_Operation_to_retrieve_list_of_subscriptions
+  Scenario: Get a list of subscriptions.
+    Given the request body is not available
+    When the request "retrieveGeofencingSubscriptionList" is sent 
+    Then the response code is 200
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And each item in the response body, if any, complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_03_Operation_to_retrieve_subscription_based_on_an_existing_subscription-id
+  Scenario: Get a subscription based on existing subscription-id.
+    Given the request body is not available and path parameter "subscriptionId" is set to the identifier of an existing subscription
+    When the request "retrieveGeofencingSubscription" is sent
+    Then the response code is 200
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body complies with the OAS schema at "/components/schemas/Subscription"
+
+
+  @geofencing_subscriptions_04_Operation_to_delete_subscription_based_on_an_existing_subscription-id
+  Scenario: Delete a subscription based on existing subscription-id.
+    Given the request body is not available and path parameter "subscriptionId" is set to the identifier of an existing subscription
+    When the request "deleteGeofencingSubscription" is sent
+    Then the response code is 202 or 204
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And if response property $.status is 204 then response body is not available 
```

Comment:
Should we add?

`And if the response property $.status is 202 then response body complies with the OAS schema at "/components/schemas/SubscriptionAsync"`